vita3k/config/include/config/state.h

```diff
@@ -179,7 +179,6 @@ struct Config : YamlLoader {
         std::string screen_filter = "Bilinear";
         bool v_sync = true;
         int anisotropic_filtering = 1;
-        bool stretch_the_display_area = false;
```

Comment:
Put this, and everything related to display stretching in another PR or at least in another commit

Reply:
No need new commit for that, is just small détail

Reply:
It's not a small detail, it's half the lines changed by this PR, and it has nothing to do with a commit whose goal is to add an option to disable the compat warnings.

Reply:
Off course is small detail, it changes absolutely nothing to the code, it's just deleted a bool variable from the config to put it locally that's all.  I'm not going to make a commit for that, it's stupid... that's why it's well written in the commit on the second line, it's secondary.

Reply:
Well put it in another commit, it has nothing to do with this one

Reply:
I said no, because is stupid, i no like create commit for nothing
Is juste détail change, relate config, so is fine.

Reply:
> It's not a small detail, it's half the lines changed by this PR, and it has nothing to do with a commit whose goal is to add an option to disable the compat warnings.

I stand by what I said

Reply:
Is full stupid, but is done...
i ignore other so, i no want create one commit for stupid things like that, is just detail, change nothing